boards/airfy-beacon/include/periph_conf.h

```diff
@@ -109,6 +109,16 @@
 /** @} */
 
 /**
+ * @name Radio device configuration
+ *
+ * The radio is not guarded by a NUMOF define, as the radio is selected by its
+ * own module in the build system.
+ * @{
+ */
+#define RADIO_IRQ_PRIO      1
```

Comment:
You can throw this out - the priorities are all set to the same values in `cpu/cortexm_common/cortexm_init.c`...